addon/components/mobiledoc-section-attribute-button/component.js

```diff
@@ -0,0 +1,33 @@
+import { defineProperty, observer, computed } from '@ember/object';
+import Component from '@ember/component';
+import layout from './template';
+import { camelize } from '@ember/string';
+
+export default Component.extend({
+  tagName: 'button',
+  layout,
+  attributeBindings: ['type', 'title'],
+  classNameBindings: ['isActive:active'],
+  type: 'button',
+  init() {
+    this._super(...arguments);
+    this._updateIsActiveCP();
+  },
+  onForDidChange: observer('attributeName', 'attributeValue', function() {
+    this._updateIsActiveCP();
+  }),
+  _updateIsActiveCP() {
+    let attributeName = this.get('attributeName');
+    let fullPath = `editor.activeSectionAttributes.${camelize(attributeName)}`;
+    let cp = computed(fullPath, 'attributeValue', function(){
+      return (this.get(fullPath) || []).includes(this.get('attributeValue'));
+    });
+    defineProperty(this, 'isActive', cp);
+  },
+  click() {
+    let editor = this.get('editor');
+    let attributeName = this.get('attributeName');
+    let attributeValue = this.get('attributeValue');
+    editor.setAttribute(attributeName, attributeValue);
```

Comment:
I added the ability to `removeAttribute` in mobiledoc-kit so we can make sure to remove the attribute if the attribute is set to the default value (`left`). Can we incorporate that here so we don't have attributes added to things in the data when the desire is to simply go back to the browser default of `left`? It makes for a cleaner experience imo. 

I did the same thing in the react editor: https://github.com/joshfrench/react-mobiledoc-editor/pull/51/commits/d7f93b356e5ba88dbd1e5bfb6208ca2d44e7fcde#diff-d65652bc9416dc013b85e964414b9f54R13-R18

Reply:
@ZeeJab Makes sense. I'll refine accordingly.

Reply:
Updated. I took a little different approach than in the react version. I think it is better for the editor to determine the default attribute value rather than the toolbar config.